### PR TITLE
Final userguide

### PR DESCRIPTION
**Pull request by potatowagon**
Still in construction. This branch is rebased on top of current master, as of 4:41pm

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -113,6 +113,8 @@ You're at work and you crave baked potatoes. Let's do another one:
 * Notice how `bake potatoes` is above `Learn Task Tracker`. That's because
 `bake potatoes` is of number 2 priority, while `Learn Task Tracker` with an
 undefined priority takes the default lowest priority number of 0.
+
+* If you create two floating tasks with the same PRIORITY, they will be shown in order of the time they were created.
```
Comment (pyokagan) [inline docs/UserGuide.md]: @potatowagon Actually it is based on 1. Priority, 2. Lexicographical order of name. As for insertion order... depends on whether the JavaFX `SortedList` we use actually uses a stable sort, but I think just to be safe maybe fudge this point a bit.

**Comment by potatowagon** [comment]
Left inserting UI images, and autocomplete.

**Comment by potatowagon** [comment]
Left auto complete and some clean ups.
It is not ready for merge ye but please review and inform me if ive left out any commands or have errors in command format

**Comment by potatowagon** [comment]
https://github.com/potatowagon/main/blob/final_userguide/docs/UserGuide.md to view the userguide

**Comment by pyokagan** [comment]
@potatowagon Okay, I will merge this first.

**Pull request merged**